burotel/indico_burotel/client/js/overrides.js

```diff
@@ -3,12 +3,15 @@ import BootstrapOptions from './components/BootstrapOptions';
 import ExtraFilters from './components/ExtraFilters';
 import DailyButton from './components/DailyButton';
 import BurotelLanding from './components/BurotelLanding';
-
+import RowLabel from './components/RowLabel';
+import DeskInfoBoxes from './components/DeskInfoBoxes';
 
```

Comment:
doesn't eslint complain about not having two empty lines after the imports now?

Reply:
I think I have a problem with eslint/sass-lint when I work on the plugins (pre-commit not being executed?). I'll have a look tomorrow.

Reply:
Solved, I was missing a dependency in the plugins repo. It complains now, thanks!